### ISSUE TITLE
[question] using import = require() without export = 

### ISSUE THREAD

**Issue by nippur72** [Microsoft/TypeScript#9309]
The handbook says:
>When importing a module using export =, TypeScript-specific import let = require("module") must be used to import the module.

does that mean it has to be used ONLY in that case? Can I use it to load normal ES6 modules too? Eg:
```ts
// m.ts
export function foo() { ... }

// main.ts
import M = require("./m");  // instead of: import * as M from "./m";
M.foo();
```
currently TypeScript doesn't complain, so I am asking is there any reason I should not use the `import = require` syntax ?

**Comment by kitsonk** [comment]
> does that mean it has to be used ONLY in that case?

No.

These sorts of questions are best asked on StackOverflow, Gitter or IRC.

**Comment by nippur72** [comment]
ok thanks
